packages/client/components/PokerDimensionValueControl.tsx

```diff
@@ -90,19 +125,30 @@ const PokerDimensionValueControl = (props: Props) => {
   const lastServiceFieldNameRef = useRef(serviceFieldName)
   const canUpdate =
     pendingScore !== finalScore || lastServiceFieldNameRef.current !== serviceFieldName
-  useSetFinalScoreError(stageId, error)
+  const {addMissingJiraFieldModalPortal, closeAddMissingJiraFieldModal} = useHandleFinalScoreError({
+    error,
+    stageId,
+    serviceField
+  })
 
   useLayoutEffect(() => {
     setPendingScore(finalScore)
     lastServiceFieldNameRef.current = serviceFieldName
   }, [finalScore])
+
   useEffect(() => {
     if (error) {
       // we want this for remote errors but not local errors, so we keep the 2 in different vars
       setPendingScore(finalScore)
     }
   }, [error])
 
+  useEffect(() => {
+    if (finalScore) {
```

Comment:
Good point. I was thinking about this scenario but it's hard to decide what is better. There are two options:
1. Close the modal after `AddMissingJiraFieldMutation` finishes successfully, reuse the `submitScore` from `PokerDimensionValueControl` and let that component handle the errors of `PokerSetFinalScoreMutation` 
2. Show the errors within `AddMissingJiraFieldModal` the way it's done now

Now, as we're talking about it I think it's better to do 1. because `PokerSetFinalScoreMutation` are not really related to the `AddMissingJiraFieldsModal`. Any thoughts @mattkrick?

Reply:
i like option 1! Always nice to re-use patterns as much as possible